src/YouTubeToHtml5.js

```diff
@@ -474,9 +477,7 @@ YouTubeToHtml5.prototype.load = function() {
     const elements = this.getElements( this.options.selector );
 
     if ( elements && elements.length ) {
-        elements.forEach( element => {
-            this.loadSingle( element );
-        } );
+        return Promise.all(elements.map( element =>  this.loadSingle( element ) ));
```

Comment:
I think we need Promise.allSettled here, but I think we need to get babel running on this before we add it, since it's kind of new (late 2019).
```suggestion
        return Promise.allSettled(elements.map( element =>  this.loadSingle( element ) ));
```